src/main/java/com/rybka/service/ExchangeService.java

```diff
@@ -0,0 +1,100 @@
+package com.rybka.service;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.rybka.dao.HibernateDAO;
+import com.rybka.model.Currency;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ExchangeService {
+
+    private static final String API_KEY = "4e0bbdc44fcdf05612fa0882";
+
+    private Double currencyValue;
```

Comment:
Move this state to the separate object and return that object from the `loadCurrencyOf()` method. 

var data = loadCurrencyOf("UAH", "USD");

data.getCurrencyValue();
data.getAmount();

Reply:
fixed